jina/types/arrays/traversable.py

```diff
@@ -2,6 +2,11 @@
 from typing import Iterable
 
 
+def _check_traversal_path_type(traversal_paths):
```

Comment:
I prefer calling it `_assert_traversal_paths_type`. That is completely a matter of taste and I am good if you ignore this comment.